Maths/FindLcm.js

```diff
@@ -33,4 +35,19 @@ const findLcm = (num1, num2) => {
   }
 }
 
-export { findLcm }
+// Typically, but not always, more efficient
+const findLcmWithHcf = (num1, num2) => {
+  // If the input numbers are less than 1 return an error message.
+  if (num1 < 1 || num2 < 1) {
+    return 'Please enter values greater than zero.'
```

Comment:
Use errors rather than return values (also rewrite it to "Numbers must be positive" or the like - this is supposed to be a programmer-facing error, not a user interface).

Reply:
Have revised.